### PR TITLE
Center of star graph

### PR DESCRIPTION
**Pull request by SwarnimaShishodia**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Description

(Find Center of Graph using C++)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Request.

(#5079)

### CODE REVIEW

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/* There is an undirected star graph consisting of n nodes labeled from 1 to n. A star graph is a graph where there is one center node and exactly n - 1 edges that connect 
+the center node with every other node.
```
Comment (HarshCasper) [inline C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: Shorten this line for easy readibility

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp (hunk 1 above)
Comment (SwarnimaShishodia) [reply C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: > Shorten this line for easy readibility

Done.

File: C-Plus-Plus/README.md [hunk 2]
```diff
@@ -251,7 +251,9 @@
 - [Shortest Path in Unweighted Graph](graphs/shortest_path_using_bfs.cpp)
 - [Depth first Search for a graph](graphs/dfs_of_a_graph.cpp)
 - [Diameter of Binary Tree](graphs/Diameter_of_binary_tree.cpp)
+- [Childern Sum Property](graphs/Childern_Sum_Property.cpp)
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md]: Why this change here?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/README.md (hunk 2 above)
Comment (SwarnimaShishodia) [reply C-Plus-Plus/README.md]: > Why this change here?

It was by mistake. I have removed it.

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+   / \
+  1   3 
+  
+ Here 2 is the answer.
+ 
+ Approach: Store the nodes in vector that are connected to partciular node, if size of that vector equals to nodes-1 then print it is the centre. */
+
+#include <iostream>
+#include<vector>
+#include<climits>
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: replace header with `#include <bits/stdc++.h>`

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+1
+2
+5
+1
+1
+3
+1
+4
+OUTPUT:
+1
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: remove extra test case one is enough

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+Time Complexity: O(no_of_edges)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/graphs/Center_of_Star_Graph.cpp:83]: Add SC as well

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+INPUT:
+3
+1
+2
+2
+3
+4
+2
+OUTPUT:
+2
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: Use proper display message how you are reading and writing outputs

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp (hunk 6 above)
Comment (SwarnimaShishodia) [reply C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: Done all the mentioned changes.

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+    cout<<(findCenter(edges));
+    return 0;
+}
+
+
+/* 
+
+First line contain no. of edges (n)
+Then 2*n line contains edges, where first line is source and second one is destination
+
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: Remove this part .Use` cout` for proper display message in your code how you are reading and writing outputs

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/graphs/Center_of_Star_Graph.cpp (hunk 7 above)
Comment (SwarnimaShishodia) [reply C-Plus-Plus/graphs/Center_of_Star_Graph.cpp]: > Remove this part .Use` cout` for proper display message in your code how you are reading and writing outputs
Done

**Review by ankitaggarwal23 — APPROVED** [review]
